sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -211,6 +211,17 @@ static int adsp_shared_base_ioremap(struct platform_device *pdev, void *data)
 	return 0;
 }
 
+static int mt8195_run(struct snd_sof_dev *sdev)
+{
+	u32 adsp_bootup_addr;
+
+	adsp_bootup_addr = SRAM_PHYS_BASE_FROM_DSP_VIEW;
+	dev_dbg(sdev->dev, "[ADSP] HIFIxDSP boot from base : 0x%08X\n", adsp_bootup_addr);
```

Comment:
Btw, do you need these `[ADSP]` prefixes in debug prints?

Reply:
OK. I will remove it. thanks.